imports/client/components/AccountForm.tsx

```diff
@@ -177,47 +204,42 @@ const AccountForm = (props: AccountFormProps) => {
   }[props.format];
 
   const emailInput = (
-    <div className="at-input form-group">
-      <label className="control-label" htmlFor="at-field-email">Email</label>
+    <div className="form-group">
```

Comment:
I still think (someday) this would all benefit from just using react-bootstrap directly, but that needn't be now

Reply:
Aaaand literally the next commit does that.  This is what happens when I review commit-by-commit 😂